components/navbar/navbar.scss

```diff
@@ -13,6 +13,7 @@
     align-self: center;
 
     a {
+      font-family: "Open Sans";
```

Comment:
is this enough or do we need to specify fallback font

Reply:
You might want to add `"Helvetica", "Arial", sans-serif` as fallbacks.